gen_prejit.sh

```
@@ -0,0 +1,183 @@
+find "${PREJIT_DIR}" -maxdepth 1 -name 'GB*.c' -delete
+echo "   PreJIT directory cleared."
+
+# ---------------------------------------------------------------------------
+# Step 4 – Clear the GraphBLAS JIT runtime cache so kernels are regenerated
+#           fresh rather than loaded from a previous warm-up run.
+# ---------------------------------------------------------------------------
+echo "[Step 4] Clearing GraphBLAS JIT runtime cache (${SUITESPARSE_GRB})..."
+if [[ -d "${SUITESPARSE_GRB}" ]]; then
+    rm -rf "${SUITESPARSE_GRB}/tmp" \
+           "${SUITESPARSE_GRB}/c"   \
+           "${SUITESPARSE_GRB}/lib"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /lib .

(SC2115)